### PR TITLE
modifications from Rachel M

### PR DESCRIPTION
**Pull request by rmandelb**
There are a number of modifications to the text in this PR.  For the full list of what I was trying to accomplish, see https://github.com/LSSTDESC/overview_paper/issues/8.  A few notes:

- I did some minimal reference updating, in places where things were egregiously out of date.  The text would improve from more of this, but there's not a lot of time before the end of the day on Monday so I went with critical things only.

- There was (in section 4.1) an estimate of 1-2% photo-z accuracy (sigma_z/(1+z)) for the gold sample.  I thought this was supposed to be 3-5% according to the latest DESC estimates, so I changed it.  @sschmidt23 @morriscb can you please confirm?

- Section 2.1.1 had strong claims about multi-fitting being needed, whereas in practice no decisions have been made about whether certain galaxy properties will be measured in a multi-fit or stack-fit approach.  I was going to change this to be a little more vague, but I noticed that similar assertions are made in section 3.3, which is not our responsibility to update.  @drphilmarshall - do you know who is updating section 3.3?  I would like to touch base with whoever it is to make sure consistent statements end up in both places.

- Parameter constraint figures 22-24 appear to be based on out-of-date assumptions about the photo-z quality, shear sample properties, usable scales, etc. from what I can tell (some details are not actually explained on the figure).  Not sure what we can do about this on such short notice, however!

### CODE REVIEW

**Comment by jablazek** [comment]
@rmandelb : any thoughts on treatment of shear peaks or possible addition of standard sirens as a DESC-relevant observable?

**Comment by rmandelb** [comment]
@jablazek , I took care of both in this PR.  Please let me know if you have comments/concerns about the changes.

**Review by drphilmarshall — APPROVED** [review]
These changes look good to me, thanks very much @rmandelb ! Approving, but will wait for conversation to converge (eg with PZ confirmation) before merging.

**Comment by drphilmarshall** [comment]
@ivezic : See the query below from @rmandelb about Section 3.3 ("Data Products"). We're wondering about updating the galaxy measurement parts of both (or neither) Section 2.1.1 and Section 3.3. What do you think?

> Section 2.1.1 had strong claims about multi-fitting being needed, whereas in practice no decisions have been made about whether certain galaxy properties will be measured in a multi-fit or stack-fit approach. I was going to change this to be a little more vague, but I noticed that similar assertions are made in section 3.3, which is not our responsibility to update. Phil, do you know who is updating section 3.3? I would like to touch base with whoever it is to make sure consistent statements end up in both places.

**Comment by drphilmarshall** [comment]
Also, @ivezic - we're checking with our PZ leads ( @sschmidt23 @morriscb ) but we think the photo-z accuracy for gold sample should be 3-5%, not 1-2% as currently written. OK to make this change in Section 4.1? Would it affect anything else in the paper?

**Comment by damonge** [comment]
I believe there's a statement about only 10% of the sample having redshift errors beyond 4% that would also be good to check.

**Comment by ivezic** [comment]
@drphilmarshall 

It depends on what you mean by "photo-z accuracy". Are you talking about the
minimum required accuracy to deliver some science, or perhaps what accuracy 
you expect to achieve with LSST data? Whatever we say in the overview paper 
must be consistent with the SRD statements ("In order to achieve this comoving 
distance accuracy, the photometric redshifts requirements for this i < 25 flux-limited 
galaxy sample are 
i) the rms (σ) for error in (1 + z) must be smaller than 0.02, 
ii) the fraction of 3σ (“catastrophic”) outliers must be below 10%, and 
iii) the bias must be below 0.003." 

I think it should be fine to make statements about the needed accuracy to deliver
some specific science goal in the overview paper, but the overview paper is certainly 
not the place to revisit the SRD requirements (that's doable too, but that would be a 
totally different process).  I also note that *even if* photo-z experts think that we cannot 
achieve 2% photo-z (e.g. because of stronger galaxy evolution effects than thought 
previously), it is not obvious that the SRD should be updated because those photo-s
performance values above were used to derive photometric precision requirements 
using a *well-defined* model-based galaxy SED set. 

Btw, I already contacted Robert et al. regarding the Multifit question from  @rmandelb

**Comment by drphilmarshall** [comment]
@rmandelb Re figures 22-24: How about we retain Fig 22 (contours) and Fig 24 (fractional errors on distance and growth of structure) from the Science Book, but comment out Figure 23 (galaxy power spectrum, with old assumptions written on the figure). Reducing the number of figures will not be a bad thing, I think. We would need to adjust the text not to refer to Fig 23, and explain the assumptions going into Fig 22 and 24. What do you think?

**Comment by rmandelb** [comment]
@ivezic - thanks for pointing out the need for consistency with the LSST SRD.  I have reverted a change that would violate this requirement.

Currently the DESC's estimates of photo-z quality do not meet what's written in the text of this paper as far as I know, but I guess we should not do anything about this now?  Would it be fair to make the text clearer that this is not the expected photo-z quality including all nuisance systematics, but rather a statement about photo-z quality given the photometric measurement quality and a set of fixed assumptions for given/known templates?

Thanks for checking about the multifit question.

**Comment by drphilmarshall** [comment]
OK, thanks @ivezic : we'll find a better way to address the photo-z's. I see Rachel is on the case! :-)

**Comment by ivezic** [comment]
@rmandelb
I think it would be fine to add this clarification about photo-z performance 
("this is not the expected photo-z quality including all nuisance systematics...").

Where can I find more quantitative details about the claim of "3-5%" performance?
Although I'm not a photo-z expert, based on published papers I can hardly believe 
that rms of 5% could be correct!

**Comment by rmandelb** [comment]
@ivezic - great, I will add suitable caveats (and try to make sure that what number goes in is suitably backed up/justified).

@drphilmarshall - I will take a stab at what you suggest for the figures, but it'll be later today.

**Comment by sschmidt23** [comment]
As @ivezic said, it really depends on what you mean by "photo-z accuracy", and how you define your statistics.  An obvious example can be seen in Melissa Graham's paper analyzing the impact of survey strategy on photo-z (link [here)](https://arxiv.org/abs/1706.09507).  This paper used a simple photo-z estimator to compare different strategies, and is meant mainly to compare relative performance, as the algorithm is not as sophisticated as current state of the art methods.  But, even this method is technically able to meet the requirements as they are specifically defined in the SRD. The requirement of rms<0.02(1+z) in the SRD refers to the *entire* 0.3<z<3.0 sample.  If you look at Figure 2, you can see that in tomographic bins at z>1.0 have RMS>0.02, but most of the galaxies are at z<1.0 and the ensemble still has an overall RMS <0.02(1+z), (with similar performance for bias and cat.outlier rate as well).  So, the difference between overall distributions and tomographic bins is one common reason for confusion in whether or not we are meeting metrics.

On the actual performance that we expect, 0.02(1+z) is optimistic: remember that the Science Book and Melissa's work both assume perfect knowledge of training sets/galaxy populations and don't deal with effects like blending.  We don't yet know exactly how representative our training sets will be, so that is hard to quantify, though we will be testing different models of incomplete training sets and more realistic photometry effects in DC2. But, given other results on real observational data I think the 3-5% range is perhaps a bit more realistic, particularly in a tomographic bin at higher redshift.

**Comment by drphilmarshall** [comment]
Thanks Sam - this is helpful. I'll push a small change making clear the
meaning of 0.02.

**Comment by jablazek** [comment]
@rmandelb @drphilmarshall : the changes regarding shear peaks and standard sirens look good. The first 3 bullet points in sec 4.1 read a bit strangely to me, when taken together. Points (2) and (3) discuss gg lensing and gg clustering individually, but shear-shear does not get its own bullet. I could try some minor edits here, if desired, but don't want to interfere with the intended structure.

**Comment by drphilmarshall** [comment]
I say go for it! We'll focus on the figures here. Thanks @jablazek.

**Comment by rmandelb** [comment]
@sschmidt23 @drphilmarshall - I updated the text on photo-z, citing Newman+15 and Graham+17.

@jablazek - Regarding this:
> Points (2) and (3) discuss gg lensing and gg clustering individually, but shear-shear does not 
> get its own bullet.

My reasoning was this: if you consider gg lensing and gg clustering on their own, they can be used for additional interesting measurements beyond the 3x2pt if you use them on different scales or with other observables.  But shear-shear lives pretty much only in the 3x2pt context, so it doesn't have a separate bullet.

**Comment by rmandelb** [comment]
@drphilmarshall - I just pushed a commit with some figure updates.  I commented out 23, and updated the caption of 22 to comment on the impact of changes in survey parameters since the LSST science book.  For the figure formerly known as 24 (now 23), it's less clear how to deal with this without actually recalculating stuff.

**Comment by ivezic** [comment]
@rmandelb @drphilmarshall 

Update on Multifit:
DM did decide that we will use Multifit and that’s why it's in our 
official Project baseline. Until we change the baseline, Multifit it is. 
Of course, we can note in the overview paper that ongoing work hints
that Stackfit might suffice (as Tony has been arguing for years…). 
Indeed, Robert already added the following sentence (in the context
of Multifit): "It is not yet clear to what extent we will be able to make some 
of these measurements on suitable linear combinations of input images 
(with careful propagation of PSFs and noise)."

**Comment by rmandelb** [comment]
@ivezic - thanks for the update.

**Review by drphilmarshall — APPROVED** [review]
Thanks very much, Rachel! I think the treatment of the figures is good enough for this paper.
